Gemfile

```diff
@@ -34,6 +34,6 @@ group :development do
   gem 'netrc', require: false
   gem 'octokit', require: false
   gem 'pry-rescue'
-  gem 'rubocop', '~> 0.49.0' # TODO: should match Gemfile HoundCi
+  gem 'rubocop', '0.91.0'
```

Comment:
We can update to `1.2.0`, I think there will be not much changes as I have checked in my local machine.

Reply:
Currently, the [highest supported by hound](http://help.houndci.com/en/articles/2461415-supported-linters) is 0.91.0.

Reply:
We can use something else than hound in that case? Github actions like [lint-action](https://github.com/marketplace/actions/lint-action).

Reply:
@KapilSachdev That sounds reasonable. So that we don't bog this Issue down, can you file a separate issue for that?

Reply:
The time i commented about `lint-action`, i was unaware of one issue that they have i.e. annotations don't work when PR is created from a fork. Which is where most PRs come from. Though there is a new `pull_request_target` event for that, but it still isn't working (from what i had tried).
`shoulda-matcher` gem also recently reverted from `lint-action` (which initially uses hound) and has fallen back to non-annotation based `rubocop` linting (like what rails uses).

I will create a issue for `rubocop` linting based on github actions.